app/views/_includes/filter-panel.njk

```diff
@@ -190,6 +190,30 @@
         }
       ]
     }) }}
+
+    {{ govukCheckboxes({
+      idPrefix: 'studyMode',
+      name: 'studyMode',
+      classes: "govuk-checkboxes--small",
+      fieldset: {
+        legend: {
+          text: 'Study mode',
```

Comment:
As above

Reply:
Changed to "Full time or part time" as suggested.